server/models/payment.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+
+const paymentSchema = new mongoose.Schema(
+  {
+    sitterId: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    userId: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    stripeCustomerId: {
+      type: String,
+      required: true,
+    },
+    rate: {
+      type: Number,
+      required: true,
+    },
+    hoursOfService: {
+      type: Number,
+      required: true,
+    },
+    paid: {
+      type: Boolean,
+      default: false,
+    },
+    cancel: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  { toJSON: { virtuals: true } }
```

Comment:
Hey @MUGABA, good schema implementation. But I don't if you are planning to use this virtual in any case. Not bad though. :+1:

Reply:
Since the value of `totalPayment` is computed I thought virtuals will be fit for it. I will see whether there is a better option.Thanks